core/dictgen/src/TModuleGenerator.cxx

```diff
@@ -445,45 +494,29 @@ void TModuleGenerator::WriteRegistrationSource(std::ostream &out,
                   inlinedHeaders + "\n"
                   "#undef  _BACKWARD_BACKWARD_WARNING_H\n";
 
-   // Dictionary initialization code for loading the module
-   out << "namespace {\n"
-       "  void TriggerDictionaryInitialization_"
-       << GetDictionaryName() << "_Impl() {\n"
-       "    static const char* headers[] = {\n";
-   if (fInlineInputHeaders) {
-      out << 0 ;
-   } else {
-      WriteHeaderArray(out);
-   };
+   std::string headerArray = "0";
```

Comment:
Should this be `= "nullptr"`?

Reply:
Yes, it should, however, the makepch.py has a regex based on this and I failed to adapt it...